cluster/common.sh

```diff
@@ -630,6 +630,7 @@ ENABLE_MANIFEST_URL: $(yaml-quote ${ENABLE_MANIFEST_URL:-false})
 MANIFEST_URL: $(yaml-quote ${MANIFEST_URL:-})
 MANIFEST_URL_HEADER: $(yaml-quote ${MANIFEST_URL_HEADER:-})
 NUM_NODES: $(yaml-quote ${NUM_NODES})
+CLUSTER_MASTER_HOSTS: $(yaml-quote ${MASTER_NAME})
```

Comment:
How about `INITIAL_ETCD_CLUSTER`? I think it will better describe that it's about a) etcd and b) that it's only an initial list. WDYT?